### PR TITLE
Input option to argument (click, #100)

### PR DESCRIPTION
**Pull request by hrshdhgd**
refactoring `@click.option` to `@click.argument`

### CODE REVIEW

File: sssom/parsers.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ def from_tsv(
     """
     if validators.url(file_path) or os.path.exists(file_path):
         df = read_pandas(file_path)
+        if 'confidence' in df.columns:
```
Comment (matentzn) [inline sssom/parsers.py:40]: why is this change here?

File: tests/data/basic.tsv [hunk 2]
```diff
@@ -27,7 +27,7 @@ x:bone_element	bone element	owl:equivalentClass		z:bone_tissue		Lexical	x	z	rdf_
 x:bone_element	bone element	owl:equivalentClass		z:bone_tissue		Lexical|Stemming	x	z	rdf_matcher	0.2	oio:hasBroadSynonym	oio:hasBroadSynonym	biolink:AnatomicalEntity	biolink:AnatomicalEntity	bone	.
 x:bone_tissue	bone tissue	owl:equivalentClass		y:bone	bones	Lexical|Stemming	x	y	rdf_matcher	0.534601961	oio:hasBroadSynonym	rdfs:label	biolink:AnatomicalEntity	biolink:AnatomicalEntity	bone	.
 x:bone_tissue	bone tissue	owl:equivalentClass		z:bone_element		Lexical	x	z	rdf_matcher	0.261203875	oio:hasBroadSynonym	oio:hasBroadSynonym	biolink:AnatomicalEntity	biolink:AnatomicalEntity	bone	.
-x:bone_tissue	bone tissue	owl:equivalentClass		z:bone_element		Lexical|Stemming	x	z	rdf_matcher	0.2	oio:hasBroadSynonym	oio:hasBroadSynonym	biolink:AnatomicalEntity	biolink:AnatomicalEntity	bone	.
+x:bone_tissue	bone tissue	owl:equivalentClass		z:bone_element		Lexical|Stemming	x	z	rdf_matcher		oio:hasBroadSynonym	oio:hasBroadSynonym	biolink:AnatomicalEntity	biolink:AnatomicalEntity	bone	.
```
Comment (matentzn) [inline tests/data/basic.tsv:30]: How is this test file change related to the PR?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: sssom/parsers.py (hunk 1 above)
Comment (hrshdhgd) [reply sssom/parsers.py:40]: Apparently `msdf.df` from some test tsvs do not have a 'confidence' column. If this makes no sense I can dig deeper.

File: tests/data/basic.tsv (hunk 2 above)
Comment (hrshdhgd) [reply tests/data/basic.tsv:30]: Basically changed one of the rows to have no confidence `''` blank.

File: sssom/parsers.py (hunk 1 above)
Comment (matentzn) [reply sssom/parsers.py:40]: I dont understand what this has to do with the input argument ticket :), that's all.

File: tests/data/basic.tsv (hunk 2 above)
Comment (matentzn) [reply tests/data/basic.tsv:30]: thats cool, just didnt understand how this relates to the scope of this PR

File: sssom/parsers.py (hunk 1 above)
Comment (hrshdhgd) [reply sssom/parsers.py:40]: ohhh ... so basically when I switched branches, for some reason changes from #100  also got included in spite of those changes being on a different branch. Have you seen experienced something like this before? Am I doing something wrong to cause this?

Comment (matentzn) [reply sssom/parsers.py:40]: Probably just an accident being on the wrong branch when running git checkout -b..

Comment (hrshdhgd) [reply sssom/parsers.py:40]: Possible ...sorry about that.

**Review by matentzn — APPROVED** [review]
